conftest.py

```diff
@@ -79,6 +79,12 @@ def skipif(items, whole_module=False):
            isinstance(configuration['platform'], Cpu64):
             skipit = "`icx+cpu64` won't work with this test"
             break
+        # Skip if icx or advisor are not available
+        if i not in 'only-advisor' or \
```

Comment:
this is wrong, it either has to be a tuple or something's off

Reply:
yes